chains/evm/contracts/interfaces/IPoolV1V2.sol

```diff
@@ -0,0 +1,7 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IPoolV1} from "./IPool.sol";
+import {IPoolV2} from "./IPoolV2.sol";
+
+interface IPoolV1V2 is IPoolV1, IPoolV2 {}
```

Comment:
Split the interface so people can implement just v2 once 1.7 is live on all chains